erlang_ls.config

```diff
@@ -0,0 +1,10 @@
+apps_dirs:
```

Comment:
I would not add this file to version control. Many users (including myself) customize it according to their own setup. The need for setting paths should disappear once we get the BSP part finished. Maybe include it as a `erlang_ls.config.sample`?

Reply:
Nit: Also, indentation is off for the two lists.

Reply:
Fixed as a follow up.